src/__tests__/Search/fetchRevisionsBySearch.test.tsx

```diff
@@ -50,11 +50,69 @@ describe('SearchView/fetchRevisionsByAuthor', () => {
     ).toBeInTheDocument();
   });
 
+  it('should fetch revisions by bug number', async () => {
+    const { testData } = getTestData();
+    (global.fetch as FetchMockSandbox).get(
+      'glob:https://treeherder.mozilla.org/api/project/*/push/*',
+      {
+        results: testData,
+      },
+    );
+
+    // set delay to null to prevent test time-out due to useFakeTimers
+    const user = userEvent.setup({ delay: null });
+
+    await renderSearchViewComponent();
+
+    expect(screen.getAllByText('try')[0]).toBeInTheDocument();
+    const searchInput = screen.getAllByRole('textbox')[0];
+    await user.type(searchInput, '123456');
+    act(() => void jest.runAllTimers());
+    expect(global.fetch).toHaveBeenCalledWith(
+      'https://treeherder.mozilla.org/api/project/try/push/?search=123456',
+      undefined,
+    );
+
+    await screen.findAllByText('Bug 123456 Fuzzy She turned me into a newt!');
+    expect(
+      screen.getAllByText("it's just a flesh wound")[0],
+    ).toBeInTheDocument();
+  });
+
+  it('should fetch revisions by phrase', async () => {
+    const { testData } = getTestData();
```

Comment:
By `phrase` you mean `comments`, yes? If so, lets use the same language to stay consistent: comments